v2/emojis/face_sad.json

```diff
@@ -1,39 +1,30 @@
 {
   "blacklist": "",
   "emojis": {
-    "\uE000": ["expressionless", "closed_eyes"],
-    "\uE001": ["diagonal_mouth"],
-    "\uE002": ["really?", "disappointed"],
-    "\uE003": ["disgusted", "drama"],
+    "\uE000": ["pensive", "exhausted"],
+    "\uE001": ["expressionless"],
+    "\uE002": ["diagonal_mouth"],
+    "\uE003": ["depressed", "disappointed"],
     "\uE004": ["persevere"],
-    "\uE005": ["pensive"],
-    "\uE006": ["unamused"],
-    "\uE007": ["frown", "sad"],
+    "\uE005": ["dead", "xP"],
+    "\uE006": ["stressed"],
+    "\uE007": ["neutral_face"],
 
-    "\uE008": ["shocked"],
-    "\uE009": ["guilty"],
-    "\uE00A": ["tear_smile"],
-    "\uE00B": ["tear_frown"],
-    "\uE00C": ["worry", "fearful"],
```

Comment:
was `:worry:` / `:fearful:` intentionally or accidentally removed? its absence is making me sad